src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html

```diff
@@ -10,6 +10,10 @@
 			appTelemetryInteract [telemetryInteractEdata]="learnMenuIntractEdata" routerLink="/learn">
 			{{resourceService?.frmelmnts?.tab?.courses}}
 		</a>
+		<a class="item py-8 pl-0 pr-24" [class.active]="router.isActive('/search/Groups') || router.isActive('/groups')"
```

Comment:
what is this **/search/Groups** route ?

Reply:
if groups can search then it should navigate to this route, I checked other menus and the same way I added.